samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -91,6 +91,83 @@ To bring the accessory back to its initial state and make provisioning available
 After the factory reset operation, the Fast Pair discoverable advertising gets automatically disabled to prevent a restart of the Fast Pair procedure.
 You need to press a button to set your accessory in the Fast Pair discoverable advertising mode and make it available for the next FMDN provisioning.
 
+.. _fast_pair_locator_tag_fp_adv_policy:
+
+Fast Pair advertising policy
+============================
+
+The Fast Pair advertising is managed by the sample.
+
+The Fast Pair advertising will stay turned on if at least one of the following triggers requested it:
+
+   * UI trigger - On the button action.
+   * FMDN provisioning trigger - On Account Key write operation during FMDN provisioning operation.
+   * Beacon clock synchronization trigger - After the system bootup if the device is provisioned.
+   * DFU mode trigger - On the DFU mode change.
+
+If the Fast Pair advertising is enabled, the sample will automatically select the correct advertising mode:
+
+   * Fast Pair discoverable advertising - without an Account Key stored on the device.
+   * Fast Pair not discoverable advertising - with an Account Key stored on the device.
+
+Removing one of the trigger requests does not automatically mean that the Fast Pair advertising will be disabled, as it requires the removal of all trigger requests.
```

Comment:
Noted - thanks for the additional context, the previous suggestion clearly affects the meaning then. 
But maybe it would make sense to add some context here instead? For example:

``To fully disable Fast Pair advertising, all trigger requests must be removed. 
However, not all triggers can be manually disabled by the user, as the FMDN provisioning and Beacon clock sync triggers are managed by the sample and are required for it to work correctly.
``

Reply:
New proposal works for me, replaced "sync" with "synchronization" to match the list few lines above and added "... managed by the sample *automatically* ..." to emphasize that it is done without user interaction.
Please verify if it is fine for you.